src/SUMMARY.md

```diff
@@ -29,9 +29,8 @@
 - [Committed Oblivious Transfer](./mpc/committed_ot.md)
 - [Oblivious Transfer](./mpc/oblivious_transfer.md)
 
+# Developer resources
 
-# Specification
+- [Quick Start Guide](./developers/quick_start_guide.md)
```

Comment:
I think it would be good to include the quick start guide somewhere in the beginning of the table of contents and not in the end.